### PR TITLE
Trade History

### PR DESCRIPTION
**Pull request by eswarasai**
##### Description
Implemented the trade history table UI based on the design

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [ ] Tests coverage hasn't decreased

##### Refers/Fixes
Fixes: #353

### CODE REVIEW

**Comment by nitinrgupta** [comment]
@eswarasai do you have any plan about the coverage decreased?

**Comment by eswarasai** [comment]
@nitinrgupta -- Nope

File: src/components/SimExchange/TradeHistory/Columns.js [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
     title: 'Price',
     dataIndex: 'price',
     key: 'price',
-    width: 120
+    width: 120,
+    render: (text, record) => {
+      return (
+        <span
+          className={
+            record.filledQty.indexOf('-') === -1 ? 'price buy' : 'price sell'
```
Comment (nitinrgupta) [inline src/components/SimExchange/TradeHistory/Columns.js]: @eswarasai - `filledQty`, `price` and others are in the form of `BigNumber` as per the response JSON logged.
screen shot 2018-09-17 at 12 34 55 pm

The app is breaking for using the `indexOf` on `BigNumbers`. I'm seeing this in the exisitng contract: `BTC_USDT_BIN_1539271907_FUSDT`.

**Comment by nitinrgupta** [comment]
@eswarasai are we not displaying the timestamp for the fills in trade history?

**Comment by eswarasai** [comment]
@nitinrgupta -- Ref: https://github.com/MARKETProtocol/MARKET.js/issues/186

**Pull request merged**